api/database_snapshot_types.go

```diff
@@ -25,21 +25,21 @@ type DatabaseSnapshotSpec struct {
 	SnapshotSpec `json:",inline,omitempty"`
 }
 
-type SnapshotStatus string
+type SnapshotPhase string
 
 const (
 	// used for DatabaseSnapshots that are currently running
```

Comment:
These comments are not following `go vet's` convention. Go vet expects comments for exported identifier to be like:
"<NAME-OF-IDENTIFIER> blah blah".